src/openrct2/localisation/StringIds.h

```diff
@@ -3911,8 +3911,10 @@ enum
 
     STR_COPY_INPUT_TO_CLIPBOARD = 6363,
 
+    STR_WINE_NOT_RECOMMENDED = 6368,
```

Comment:
```suggestion
    STR_WINE_NOT_RECOMMENDED = 6364,
```
To reflect the suggestion above.